Manifest.toml

```diff
@@ -0,0 +1,15 @@
+# This file is machine-generated - editing it directly is not advised
```

Comment:
Generally, Manifest.toml files for packages should not be checked into Git but rather listed in .gitignore.